requirements.txt

```diff
@@ -2,4 +2,4 @@ numpy
 rdflib
 rdfextras
 rdflib-jsonld
-pandas>=0.18.1
```

Comment:
The viewer was not working with the version of panda I had on my machine that's why I added this version constraint. That's fine with me if you want to remove it but would be best to know which version is needed.